src/main/clm_instMod.F90

```diff
@@ -433,6 +437,8 @@ subroutine clm_instInit(bounds)
        call crop_inst%InitAccBuffer(bounds)
     end if
 
+    call fates_fire_data_inst%InitAccBuffer(bounds)
```

Comment:
Same thing here. Although here this call can probably be inside of another clmfates_interface subroutine.